v3/src/components/graph/components/graph-component.tsx

```diff
@@ -33,6 +33,8 @@ export const GraphComponent = observer(({broker}: IProps) => {
   const dataset = broker?.selectedDataSet || broker?.last
   const dotsRef = useRef<SVGSVGElement>(null)
   const [showInspector, setShowInspector] = useState(false)
+  const [showParentToggles, setShowParentToggles] = useState(false)
+  const [showMeasuresForSelection, setShowMeasuresForSelection] = useState(false)
```

Comment:
These two properties will be part of the model, right, since they need to be saved and restored?

Reply:
I agree with Bill's comment that it would make more sense to put these properties in the`GraphModel` now rather than have them as React state now and move them to the `GraphModel` later.

Reply:
I was just adding the state management, but don't have to be added right now. They do have to be part of the model.